.gitignore

```diff
@@ -2,4 +2,5 @@ _site
 _assets/
 node_modules
 .sass-cache
+node_modules
```

Comment:
This is not relatable with current issue. Please separate this into different pull request.